scripts/sbatch.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import json
+import subprocess
+import argparse
+import getpass
+from enum import Enum
+
+MB_PER_GB = 1024.0
+
+DEFAULT_NTASKS = 1
+DEFAULT_CPUS_PER_TASK = 1
+
+MAHUIKA_SLURM_CONF = (
+    "/scale_wlg_persistent/filesets/home/slurm/mahuika/etc/opt/slurm/slurm.conf"
+)
+MAUI_SLUM_CONF = "/etc/opt/slurm.conf"
+
```

Comment:
I dont think we need to differentiate slurm conf for maui/mahuika.
as the slurm we currently have is compiled to read a default location, and symbolic linked to different file based on machines, set both to "/etc/opt/slurm/slurm.conf" should be enough.
( running "realpath /etc/opt/slurm/slurm.conf" on both machine will show you the actual file each machine is reading)

Reply:
What about cross platform submission?

Reply:
totally forgot about that. in that case, we  have to hard-coded it, but have to update everytime when they decided to change the config.
There isnt a good way i can think of to get this dynamically, since each machine can only see certain file system.